### ISSUE TITLE
kola: s390x: test rpmostree.upgrade-rollback fails

### ISSUE THREAD

**Issue by LakshmiRavichandran1** [coreos/fedora-coreos-tracker#1205]
# Bug Report #
on May 20 11:01:04 CEST 2022 with build fedora-coreos-36.20220518.dev.0-qemu.s390x.qcow2

```
$ cosa kola run --rerun --parallel 4 --output-dir tmp/kola
...
--- FAIL: rpmostree.upgrade-rollback (40.39s)
    --- FAIL: rpmostree.upgrade-rollback/upgrade (15.11s)
```

## Environment ##
What operating system is being used to run coreos-assembler?

```
$ cat /etc/os-release
NAME="Red Hat Enterprise Linux"
VERSION="8.4 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.4"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.4 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8.4:GA"
```

What operating system is being assembled?
fedora-coreos

Is coreos-assembler running in Podman or Docker?
podman

If Podman, is coreos-assembler running privileged or unprivileged?
as non-root user

## Expected Behavior ##
test rpmostree.upgrade-rollback should Pass

## Actual Behavior ##
test rpmostree.upgrade-rollback fails

## Reproduction Steps ##

1. bring up a s390x KVM lpar, prepare a non-root user, do cosa fetch, cosa build as per docs
2. run 'cosa kola run --rerun --parallel 4 --output-dir tmp/kola' and see the test fail.

## Other Information ##
[console.txt](https://github.com/coreos/coreos-assembler/files/8738097/console.txt)
[journal.txt](https://github.com/coreos/coreos-assembler/files/8738098/journal.txt)

**Comment by dustymabe** [comment]
Potential upstream fix: https://github.com/ostreedev/ostree/pull/2614

**Comment by LakshmiRavichandran1** [comment]
This issue should be fixed in `ostree-2022.4-2.fc36`. A PR to fast track this update was merged in https://github.com/coreos/fedora-coreos-config/pull/1778
